samza-core/src/main/java/org/apache/samza/zk/ScheduleAfterDebounceTime.java

```diff
@@ -48,7 +48,7 @@
   // Action name when the Processor membership changes
   public static final String ON_PROCESSOR_CHANGE = "OnProcessorChange";
 
-  public static final int DEBOUNCE_TIME_MS = 2000;
+  public static final int DEFAULT_DEBOUNCE_TIME_MS = 2000;
```

Comment:
Can you move this definition to JobConfig or ZkConfig? It looks like ZkJobCoordinator is the only place accessing this value. 

If we move to either of the config classes, we can avoid referring to the default value in Line 84 of ZkJobCoordinator. It will be encapsulated within the config class and the api will jobconfig.getDebounceTimeMs().